sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Create.java

```diff
@@ -586,9 +586,10 @@ public T apply(TimestampedValue<T> input) {
       throws CannotProvideCoderException {
     checkArgument(
         !Iterables.isEmpty(elems),
-        "Elements must be provided to construct the default Create Coder. To Create an empty "
-            + "PCollection, either call Create.empty(Coder), or call 'withCoder(Coder)' on the "
-            + "result PTransform");
+        "This Create class cannot deduce a Coder because it has zero " +
+           "elements.  Either add elements, or call Create.empty(Coder), or" +
+           " call 'withCoder(Coder)' on the PTransform returned from Create" +
```

Comment:
The 'Create.empty(TypeDescriptor)' and 'Create.withType(TypeDescriptor)' are also valid.